kornia/color/gray_core.py

```diff
@@ -0,0 +1,232 @@
+from __future__ import annotations
+
+import jax.numpy as jnp
+import keras_core as keras
+import numpy as np
+import tensorflow as tf
+import torch
+
+
+def KORNIA_CHECK_IS_TENSOR(x: object, msg: str | None = None, raises: bool = True):
+    if not isinstance(x, [tf.Tensor, torch.Tensor, np.ndarray, jnp.array]):
+        if raises:
+            raise TypeError(f"Not a Tensor type. Got: {type(x)}.\n{msg}")
+        return False
+    return True
+
+
+def bgr_to_rgb(image):
+    if not isinstance(image, [tf.Tensor, torch.Tensor, np.ndarray, jnp.array]):
+        raise TypeError(f"Input type is not a Tensor. Got {type(image)}")
+
+    if len(image.shape) < 3 or image.shape[-3] != 3:
+        raise ValueError(f"Input size must have a shape of (*, 3, H, W).Got {image.shape}")
+
+    # flip image channels
+    out = keras.ops.flip(image, axis=-3)
```

Comment:
Is it needed here the Keras convert_to_tensor?

Reply:
Don't think so, it should handle it anyway. Image here would assume that it's a tensor of the backend anyway